tensorflow_datasets/core/dataset_builder.py

```diff
@@ -266,22 +295,26 @@ def info(self):
     return self._info()
 
   @api_utils.disallow_positional_args
-  def download_and_prepare(self, download_dir=None, download_config=None):
+  def download_and_prepare(
+    self,
+    download_dir: str = None,
```

Comment:
If the argument can be none, please use `Optional[str]`. Here and elsewhere

Reply:
Done